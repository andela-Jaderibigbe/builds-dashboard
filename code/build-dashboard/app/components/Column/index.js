import React from 'react';
import PropTypes from 'prop-types';
import styles from './Column.scss';

const Column = ({ className, children }) => {
  className = `${styles.column} ${className}`;
  return <div className={className}>{children}</div>;
};

Column.propTypes = {
  className: PropTypes.string,
  children: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
};

export default Column;
