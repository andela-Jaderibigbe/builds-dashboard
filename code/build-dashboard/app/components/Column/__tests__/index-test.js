import React from 'react';
import renderer from 'react-test-renderer';
import Column from '../';

const content = <div className="unique" />;

it('renders correctly', () => {
  const tree = renderer.create(<Column>{content}</Column>).toJSON();
  expect(tree).toMatchSnapshot();
});
