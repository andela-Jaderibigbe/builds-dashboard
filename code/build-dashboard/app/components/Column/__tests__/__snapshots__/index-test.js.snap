// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="column undefined"
>
  <div
    className="unique"
  />
</div>
`;
